### ISSUE TITLE
Option for remember sort and view per folder

### ISSUE THREAD

**Issue by regs01** [doublecmd/doublecmd#605]
That would be really time savor. Some folder like need to be always thumbnail and sorted by date, some brief and sorted by name. So would be great to have an option to remember settings for each folder where it was set.

**Comment by vagra** [comment]
Yes, I really need this feature. This feature is available in Directory Opus, but not in DoubleCmd. The suggested solution is to add directories that need specific sorting to the favorites list and then sort them. However, this does not solve my scenario:

First, I have a root directory that needs to be sorted by time.

Then, under this root directory, I constantly add new subdirectories. There are many subdirectories and the number keeps increasing, and each subdirectory should be sorted by name.

No matter if I enter a subdirectory or return to the root directory, the directories should be sorted according to their remembered sort settings.

This work scenario means that I can easily see which new subdirectories have been added under the root directory, and then enter each new subdirectory and operate on their files. Since batch selection is often required, they need to be sorted by name.

**Comment by corpix** [comment]
As an experiment I have added a new setting prototype to `Files views`. Currently it only support saving sort column settings, not sure, maybe will do more.

<img width="1028" height="891" alt="Image" src="https://github.com/user-attachments/assets/d5221cd4-ba47-472a-ab58-703c6fdfcccb" />

Feel free to try and/or use the code
https://github.com/corpix/doublecmd/commit/d5283a8f28282a0d5a5eb686b11d39cb95b396c8

**UPD**:
per-directory view saving https://github.com/doublecmd/doublecmd/commit/694eaf83dee42e4f49952602c85f3223b9fe49c0
clean up procedure for deleted directories settings https://github.com/doublecmd/doublecmd/commit/dfdfb61b04ac45b523a7e13a7140d410d847f571

I am still testing this, may do a pull request in future

**Comment by lailoken** [comment]
I really miss this coming from Total Commander as well. I often need to change to my Downloads folder, and in Total Commander that would automatically change sort by creation time.  And leaving it again reverts to previous layout/sort.
Which makes sense! Now I constantly have to switch it manually, and then I forget to return it, causing me to get confused the next time I use it outside Downloads.
